test/epgsql_SUITE.erl

```diff
@@ -847,6 +848,17 @@ array_type(Config) ->
         Select(jsonb, [<<"{}">>, <<"[]">>, <<"1">>, <<"1.0">>, <<"true">>, <<"\"string\"">>, <<"{\"key\": []}">>])
     end).
 
+record_type(Config) ->
+    Module = ?config(module, Config),
+    epgsql_ct:with_connection(Config, fun(C) ->
+        {ok, _, [{{1, 2}}]} = Module:equery(C, "select (1,2)", []),
+        {ok, _, [{{1, {2, 3}}}]} = Module:equery(C, "select (1, (select (2,3)))", []),
+        {ok, _, [{{1, [2, 3]}}]} = Module:equery(C, "select (1, '{2,3}'::int[])", []),
```

Comment:
May you also, please, try to use `?assertMatch` instead of just pattern-matching?